awl/mslider.cpp

```diff
@@ -234,15 +234,22 @@ void MeterSlider::paintEvent(QPaintEvent* ev)
    	p.setFont(f);
       p.setPen(QPen(Qt::darkGray, 2));
    	QFontMetrics fm(f);
-      int xt = 20 - fm.width("00") - 5;
+      int xt = 20 - fm.width("-00") - 5;
 
       QString s;
-   	for (int i = 10; i < 70; i += 10) {
-      	h  = y1 + lrint(i * mh / range);
-         	s.setNum(i - 10);
-  		p.drawText(xt,  h - 3, s);
-		p.drawLine(15, h, 20, h);
-         	}
+      for (int i = minValue(); i <= maxValue(); i += 10) {
+            h  = y1 + lrint((maxValue() - i) * mh / range);
+            s.setNum(i);
+            if (i > 0)
+                  p.drawText(QRect(xt - 10, h - 4, 20, 8), Qt::AlignRight, QString::fromStdString("+") + s);
+            else if (i == minValue()) {
+                  p.drawText(QRect(xt - 10, h - 4, 20, 8), Qt::AlignRight, QString::fromStdString("-∞"));
```

Comment:
Well C++ 11 allows it, but it seems you would have to write it as u8"-∞" to guarantee portability (I've just been reading http://stackoverflow.com/questions/13444930/is-the-u8-string-literal-necessary-in-c11).  I think I heard master is C++11...although maybe 2.1 is still not C++11 (Correct me if I'm producing misinformation).